runtime/doc/options.txt

```diff
@@ -6523,6 +6523,14 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 	   C1	    Control characters 0x80...0x9F
 
+					*'termsyncout'* *'notermsyncout'*
+'termsyncout'		boolean	(default on)
```

Comment:
I see in earlier comments that 'sync' was the original name. So we could name this 'termsync', which is more readable and also leaves the door open for potentially non-output related "sync" behavior in the future.